### ISSUE TITLE
Using wide wildcards in //css_inc can be improved

### ISSUE THREAD

**Issue by oleg-shilo** [oleg-shilo/cs-script#19]
```
dir-
    |- scriptC.cs
    \- scriptD.cs
scriptA.cs
scriptB.cs
main.cs
```
Problem: if `main.cs` includes `scriptC.cs` containing //css *.cs, the scripts from the root directory will also be included (A and B). This is not a problem as such as the current directory has the highest priority when resolving wildcards and curr dir in this case is the root.

Though improvement can be make by dynamically substituting './' with the parent directory of the script containing the directive. Thus `./*.cs` will be converted into `dir/*.cs` and `*.cs` will remain the same.

**Comment by oleg-shilo** [comment]
Implemented with 2d6e21963cd3e7440b22280287422205e4e11e85

**Comment by oleg-shilo** [comment]
Fixed in the Release v3.18.0
